File: Server/TravelJournal/src/main/java/TravelJournal/payload/response/JwtResponse.java
package TravelJournal.payload.response;

import lombok.Getter;
import lombok.Setter;

import java.util.List;
@Getter
@Setter
public class JwtResponse {
    private String token;
    private String type = "Bearer";
    private String id;
    private String login;
    private String email;
    private List<String> roles;

    public JwtResponse(String accessToken, String id, String login, String email, List<String> roles) {
        this.token = accessToken;
        this.id = id;
        this.login = login;
        this.email = email;
        this.roles = roles;
    }


}

File: Client/app/src/main/java/com/example/traveljournal/data/model/response/JwtResponse.kt
package com.example.traveljournal.data.model.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


class JwtResponse {
    @SerializedName("token")
    @Expose
    var token: String? = null

    @SerializedName("type")
    @Expose
    var type: String? = null

    @SerializedName("id")
    @Expose
    var id: String? = null

    @SerializedName("login")
    @Expose
    var login: String? = null

    @SerializedName("email")
    @Expose
    var email: String? = null

    @SerializedName("roles")
    @Expose
    var roles: List<String>? = null
}
